test/views/OneCustomsProcedureCodeViewSpec.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.ViewBaseSpec
+import forms.OneCustomsProcedureCodeFormProvider
+import messages.{BaseMessages, OneCustomsProcedureCodeMessages}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.data.Form
+import play.api.mvc.Call
+import play.twirl.api.Html
+import views.html.OneCustomsProcedureCodeView
+
+class OneCustomsProcedureCodeViewSpec extends ViewBaseSpec with BaseMessages {
+
+  private lazy val injectedView: OneCustomsProcedureCodeView = app.injector.instanceOf[OneCustomsProcedureCodeView]
+
+  private val backLink: Call = Call("GET", "url")
+
+  val formProvider: OneCustomsProcedureCodeFormProvider = injector.instanceOf[OneCustomsProcedureCodeFormProvider]
+
+  "Rendering the OneCustomsProcedureCode page" when {
+    "no errors exist" should {
+
+      val form: Form[Boolean] = formProvider.apply()
+      lazy val view: Html = injectedView(form, backLink)(fakeRequest, messages)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      s"have the correct page title" in {
+        document.title mustBe OneCustomsProcedureCodeMessages.title
+      }
+
+      "not render an error summary" in {
+        document.select("div.govuk-error-summary").size mustBe 0
+      }
+
+      "not render an error message against the field" in {
+        document.select("#value-error").size mustBe 0
+      }
+    }
+
+    "an error exists (no option has been selected)" should {
+      lazy val form: Form[Boolean] = formProvider().bind(Map("value" -> ""))
+      lazy val view: Html = injectedView(form, backLink)(fakeRequest, messages)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      "update the page title to include the error prefix" in {
+        document.title mustBe OneCustomsProcedureCodeMessages.errorPrefix + OneCustomsProcedureCodeMessages.title
+      }
+
+      "render an error summary with the correct message" in {
+        elementText("div.govuk-error-summary > div") mustBe OneCustomsProcedureCodeMessages.requiredError
+      }
+
+      "render an error message against the field" in {
+        elementText("#value-error") mustBe OneCustomsProcedureCodeMessages.errorPrefix + OneCustomsProcedureCodeMessages.requiredError
```

Comment:
```suggestion
        elementText("#value-error") mustBe errorPrefix + OneCustomsProcedureCodeMessages.requiredError
```

Reply:
Your OneCustomsProcedureCodeViewSpec is extending BaseMessages, where the errorPrefix is defined, hence your messages file doesn't need to extend it and you don't need to access it through your messages file